### PR TITLE
Upstair.Presty-Installer version 1.0.0

### PR DESCRIPTION
**Pull request by confidire-official**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99240)

### CODE REVIEW

**Comment by Upstair** [comment]
@microsoft-github-policy-service agree [company="Upstairs Co.,ltd"]

**Comment by Upstair** [comment]
> @Upstair请阅读以下贡献者许可协议 （CLA）。如果您同意CLA的观点，请回复以下信息。
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > 选项：
> > 
> > * （默认 - 未指定公司）我对我的提交内容拥有唯一的知识产权，并且我不是在为我的雇主工作的过程中提交内容。
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * （当公司给出时）我在为我的雇主工作的过程中提交内容（或者我的雇主根据合同或适用法律对我的提交内容拥有知识产权）。我已获得雇主的许可，可以代表我的雇主提交意见并签订本协议。通过在下面签名，定义的术语“您”包括我和我的雇主。
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> 贡献者许可协议

@microsoft-github-policy-service agree [company="Upstairs"]

**Comment by Upstair** [comment]
> @microsoft-github-policy-service agree [company="{your company}"]

@microsoft-github-policy-service agree

File: manifests/u/Upstair/Presty-Installer/1.0.0/Upstair.Presty-Installer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Upstair.Presty-Installer
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x86
+  InstallerType: exe
```
Comment (stephengillie) [inline manifests/u/Upstair/Presty-Installer/1.0.0/Upstair.Presty-Installer.installer.yaml:8]:
```suggestion
  InstallerType: portable
```

**Pull request closed without merging**